products.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+require('vendor/autoload.php');
+
+use FurnitureStoreAPI\Products\ProductsHydrator as ProductsHydrator;
+use FurnitureStoreAPI\DatabaseConnection\DBConnect as Connection;
+use FurnitureStoreAPI\Response\LoggingService as ErrorLogging;
+use FurnitureStoreAPI\Response\ResponseService as Response;
+use FurnitureStoreAPI\Services\Headers as SetHeaders;
+use FurnitureStoreAPI\Exceptions\InvalidCategoryException as InvalidCategoryException;
+
+SetHeaders::apiHeaders();
+
+try {
+    if (isset($_GET['cat']) && $_GET['cat'] < 12 && $_GET['cat'] > 0) {
```

Comment:
The danger of checking if the id is < 12 here is that if a new category is added, you'd have to remember to change the code here every time. I guess instead you could throw this exception from the hydrator if the query doesn't return any results.

Reply:
I think it's fine just to use `is_numeric` to check the `cat` param here instead.